testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java

```diff
@@ -43,9 +43,16 @@
 public class ResultFragment extends Fragment {
 
     public static final String CORRELATION_ID = "correlation_id";
+    public static final String ACCOUNT_TENANT_ID = "account_tenant_id" ;
     static final String ACCESS_TOKEN = "access_token";
     static final String DISPLAYABLE = "displayable";
     static final String STRING_DATA_TO_DISPLAY = "string_data_to_display";
+    public static final String TENANT = "tenant";
```

Comment:
Is this home tenant vs account tenant which is from a specific authority which may or may not be the same as the home tenant?

Reply:
To the best of my knowledge here, this is the tenant that issued the token, which may not be the same as the one specified in the authority parameter of the request.  Examining this value could show you the tenant mismatch that was seen in our recent problems.